README.md

```diff
@@ -1,48 +1,56 @@
-- Uses the best image compression algorithms available: [mozjpeg](https://github.com/mozilla/mozjpeg) and [libvips](https://github.com/libvips/libvips)
-- [Configurable](#Configuration) and extensible: use default settings or adapt to your needs
-- Runs in [GitHub Actions](https://github.com/features/actions)
-- Built by web performance experts at [Calibre](https://calibreapp.com/); a performance monitoring platform
-
-![Preview of image-actions Pull Request comment](https://user-images.githubusercontent.com/924/62024579-e1470d00-b218-11e9-8655-693ea42ba0f7.png)
-
-### Table of Contents
+Image Actions is a Github Action built by performance experts at [Calibre](https://calibreapp.com/) that automatically compresses JPEGs, PNGs and WebPs in Pull Requests. 
 
-- [Image Actions](#image-actions)
-  - [Table of Contents](#table-of-contents)
-  - [Installation](#installation)
-  - [Configuration](#configuration)
-  - [Image quality settings](#image-quality-settings)
-  - [Running just the compression](#running-just-the-compression)
-  - [Handling pull requests from forked repos](#handling-pull-requests-from-forked-repos)
-  - [Compressing images on demand or on a schedule](#compressing-images-on-demand-or-on-a-schedule)
-  - [Combined workflow](#combined-workflow)
-  - [Migrate legacy configuration](#migrate-legacy-configuration)
-  - [Local development](#local-development)
-  - [Links and Resources](#links-and-resources)
+Image Actions offers:
 
-## Installation
-
-Create the `.github/workflows/calibreapp-image-actions.yml` file with the following configuration:
+- **Fast, efficient and near-lossless compression**
+- Best image compression algorithms available ([mozjpeg](https://github.com/mozilla/mozjpeg) and [libvips](https://github.com/libvips/libvips))
+- [Ease of customisation](#Configuration): use default settings or adapt to your needs
+- Running on demand or schedule
+
+...and more!
+
+## Table of Contents
+
+- [Usage](#usage)
+- [Configuration](#configuration)
+  * [Control image quality settings](#control-image-quality-settings)
+  * [Ignore paths](#ignore-paths)
+  * [Run compression only](#run-compression-only)
+  * [Compress on demand or on schedule](#compress-on-demand-or-on-schedule)
+  * [Process Pull Requests from forked repositories](#process-pull-requests-from-forked-repositories)
+  * [Combined workflow](#combined-workflow)
+- [Migrate legacy configuration](#migrate-legacy-configuration)
+  * [Uses and `GITHUB_TOKEN` definition](#uses-and--github-token--definition)
+  * [Location and name of the configuration file](#location-and-name-of-the-configuration-file)
+  * [Listing `ignorePaths`](#listing--ignorepaths-)
+- [Contributing](#contributing)
+- [Used By](#used-by)
+- [Authors](#authors)
+- [Resources](#resources)
+- [License](#license)
+
+## Usage
+
+1. Create a `.github/workflows/calibreapp-image-actions.yml` file in your repository with the following configuration:
 
 ```yml

```

Comment:
is this basic example is still up-to-date @benschwarz?

Reply:
Yep.